### PR TITLE
Feature/auth

### PR DESCRIPTION
**Pull request by lukaszgodula**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,14 +7,17 @@
...
     "test": "ng test --watch false",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e",
+    "replace": "replace-in-file"
   },
   "private": true,
   "dependencies": {
     "@angular/animations": "^6.1.7",
+    "@angular/cdk": "^6.2.0",
```
Comment (pevel) [inline package.json:16]: Having ^ in version number may result in different package versions installed on CI server. It's more safe to specify exact version, without any modifier.

File: src/app/app.component.spec.ts [hunk 2]
```diff
@@ -14,6 +15,9 @@ describe('AppComponent', () => {
       declarations: [
         AppComponent
       ],
+      providers: [
+        { provide: AuthService, useValue: () => { } }
```
Comment (pevel) [inline src/app/app.component.spec.ts:19]: I think it makes more sense to useValue: { }. () => { } is a factory.

File: package.json (hunk 1 above)
Comment (lukaszgodula) [reply package.json:16]: Thanks, good catch, will change that :)

File: src/app/app.component.spec.ts (hunk 2 above)
Comment (lukaszgodula) [reply src/app/app.component.spec.ts:19]: Yeah, right. TBH during the time I was fixing UT I put on my list TODO: mock Auth Service  ;)

**Pull request merged**